core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -260,7 +260,7 @@ class LogManager(logDirs: Seq[File],
       logDirFailureChannel = logDirFailureChannel)
 
     if (logDir.getName.endsWith(Log.DeleteDirSuffix)) {
-      this.logsToBeDeleted.add(log)
+      this.logsToBeDeleted.add((log, time.milliseconds()))
```

Comment:
Could we add a private method that does the above statement and reuse?

Reply:
Sure. It is fixed now.